Work on first ticket

JustinSUT opened the pull request:
-basic setup
-worked on color selection
-worked on dark mode changes
-added new options for the customization of colors

JustinSUT committed:
Work on first ticket

-basic setup
-worked on color selection
-worked on dark mode changes
-added new options for the customization of colors

JustinSUT committed:
Update

update

JustinSUT committed:
Finish first ticket

Finish first ticket

JustinSUT committed:
Update first ticket

Update first ticket

julesl23 commented:
Color Customization page looks a lot better, thanks! Color picker is great in Primary section.
But there are these major issues:

1.  For secondary section the way it chooses a default starting colour from the primary is great, but as mentioned before, I want the user to also have the option  to choose the colour exactly from the colour picker like in the primary, as companies using the player will probably have specific company brand colours that they have to use.

2.  The colour customisation settings doesn’t save as you’re still using axios. You must use fabstirdb (https://github.com/Fabstir/fabstirdb) to save the state as this allows for the state to be saved with the logged in user account.

3.  After saving the colour customisation, when I go back to home screen, the button and other colours are still the same colour, nothing changed. This might be related to point 2.

4.  Less/more slider in neutrals section seems to have too little effect

JustinSUT committed:
Update fabstirdb and customize

Update fabstirdb and customize

JustinSUT committed:
reMove axios

reMove axios

JustinSUT committed:
Update frontend

Update frontend

JustinSUT committed:
Fixed multi user issue

Fixed multi user issue

JustinSUT committed:
Fixed 2 issues

- fixed codes that use customized html tags
- fixed customized color not working for header and background

julesl23 commented:
1. Remove `color_scheme` from Attributes in NFT slide over because these are parameters that are read-only once NFT is minted and cannot be changed. Move this to Profile section.

2. Background for the Input and Select boxes in `NFTSlideOverLeft.jsx` are off. Sometimes transparent, sometimes pure white. .

3. In the UI, text `aria-label=…` appears in the color customisation section in the buttons.

4. For some reason, once I set the Primary and Secondary colours and saved, every button and input box went invisible/black until I changed the Neutrals slider to more saturated and then saved again. Then the UI controls appeared again.

5. As mentioned before, the 6 utility colours have to be editable so the user can change them. Need to be able to click their button and for the color picker to come up (like the primary and secondary). Keep their default colours you have there as the starting point.

JustinSUT committed:
fixed 5 issues

fixed 5 issues
Remove color_scheme from Attributes in NFT slide over because these are parameters that are read-only once NFT is minted and cannot be changed. Move this to Profile section.

Background for the Input and Select boxes in NFTSlideOverLeft.jsx are off. Sometimes transparent, sometimes pure white. .

In the UI, text aria-label=… appears in the color customisation section in the buttons.

For some reason, once I set the Primary and Secondary colours and saved, every button and input box went invisible/black until I changed the Neutrals slider to more saturated and then saved again. Then the UI controls appeared again.

As mentioned before, the 6 utility colours have to be editable so the user can change them. Need to be able to click their button and for the color picker to come up (like the primary and secondary). Keep their default colours you have there as the starting point.

julesl23 commented:
1.  There is some fixed white background in the NFT sliding page (press '+' in gallery section) in all the input and checkboxes.
In dark mode that really stands out. This needs to change to a darker colour in dark node. Should be using one of the customisation colours.

2. Also when going into gallery section, on the right there is some fixed white background for section titled "information", that stays white when in dark mode. Should be darker colour in dark mode. Should be using one of the customisation colours.

3. There is still that bug when I set the primary and secondary colours, the Neutrals section is all black until I move the slider.

JustinSUT committed:
Update 3 errors

Update 3 errors

JustinSUT committed:
Updated git

Updated git

JustinSUT committed:
Update code

- Removed alpha

JustinSUT committed:
Saturation error fixed,

Saturation error fixed,

JustinSUT committed:
Update only bug fix

Update only bug fix

JustinSUT committed:
Fixed issue

Fixed issue
